flink-cdc-common/src/main/java/com/ververica/cdc/common/event/TableId.java

```diff
@@ -20,7 +20,19 @@
 
 import java.util.Objects;
 
-/** Unique identifier for a database table. */
+/**
+ * Unique identifier for a database table which consists of three parts: namespace, schema name and
```

Comment:
```suggestion
 * The unique identifier used to represent the path of external table or external data collection,  all external 
 system data collection could map to a dedicated {@code TableId}.
 * <ul>
 *   <li>   {@code TableId} contains at most three parts, it will be treated as namespace, schema name and
 * table name.
  *   <li>  {@code TableId} could contain two parts, it will be treated as schema name and table name.
  *  <li>   {@code TableId}  could contain only one part, it will be treated as table name.
 *
 * <p>Connectors need to establish the mapping between {@code TableId}  and external data collection object path. For
 * example,
 *
 * <ul>
 *   <li>The mapping relationship for Oracle is: (database, schema, table).
 *   <li>The mapping relationship for MySQL or Doris is: (database, table).
 *   <li>The mapping relationship for Kafka is: (topic).
```